sql/entities.py

```diff
@@ -9,9 +9,13 @@ class User(Base):
 
     id = Column('ID', Integer, primary_key=True)
     name = Column('Username', String(length=255), nullable=False)
+    role = Column('Role', String(length=50), ForeignKey('roles.ID'), default=2)
+    password = Column('Password', String(length=25), nullable=False)
 
-    def __init__(self, name):
+    def __init__(self, name, role, pwd):

```

Comment:
We need the role here like a reqiued parameter?

Reply:
1) _a_ role
2) _as_, not _like_. You are talking about meaning, not similarity